doc/source/Windows.rst

```diff
@@ -0,0 +1,116 @@
+Windows Tutorial 
+================
+
+This guide gives you a brief introduction to how volatility3 works and some demonstration on suite of plugins available from
+
+Acquiring memory
+----------------
+
+Volatility does not provide the ability to acquire memory. 
+
+.. tip:: You could use `WinPmem <https://github.com/Velocidex/WinPmem/releases/latest>`_ for collecting windows memory dump.
```

Comment:
Probably good to offer a few possibilities to avoid the suggestion of favouritism.